### PR TITLE
RHBPMS-4285: Unfinished 'No Associated Containers' screen in Rule Deployments perspective

### PR DESCRIPTION
**Pull request by karreiro**
See https://issues.jboss.org/browse/RHBPMS-4285

### CODE REVIEW

**Comment by karreiro** [comment]
@manstis Could you please take a look? :-)

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/resources/org/kie/workbench/common/screens/server/management/client/resources/i18n/Constants.properties [hunk 1]
```diff
@@ -95,9 +95,8 @@ ProcessConfigView.KieBaseName=Kie Base Name
 ProcessConfigView.KieSessionName=Kie Session Name
 ProcessConfigView.MergeMode=Merge mode (deployment descriptor)
 ProcessConfigView.RuntimeStrategy=Runtime strategy
-RemoteEmptyView.Documentation=documentation
 RemoteEmptyView.NoAssociatedContainers=No Associated Containers
-RemoteEmptyView.NoAssociatedContainersMessage=This Remote Server does not have any associated KIE Containers. To use this Remote Server, you must configure [something] inside the Remote Server.
+RemoteEmptyView.NoAssociatedContainersMessage=This Remote Server does not have any associated KIE Containers. You can create containers on this server server by adding them to corresponding server template.
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/resources/org/kie/workbench/common/screens/server/management/client/resources/i18n/Constants.properties]: Same typo "... server server..." (well, not as such a typo, more of a duplicated word!)

**Comment by karreiro** [comment]
Thank you @manstis

**Comment by manstis** [comment]
Thanks @karreiro

**Comment by manstis** [comment]
Merge when green...

**Pull request merged**